dash/Dashboard.py

```diff
@@ -228,11 +228,12 @@ def plot_map_chart(self, vehicles,pedestrians,traffic_light_states,static_object
         fig = plt.figure(Dashboard.MAP_FIG_ID, frameon=False, clear=True)
         plt.cla()
 
+        map_area = self.sim_traffic.origin[3] if len(self.sim_traffic.origin) > 3 else MPLOT_SIZE
```

Comment:
is this check ever false len(self.sim_traffic.origin) > 3? Origin always have 4 members...